macros/upload_results.sql

```diff
@@ -126,14 +126,22 @@
 
         {% do log("Uploading sources", true) %}
         {% set sources = dbt_artifacts.get_relation('sources') %}
-        {% set content_sources = dbt_artifacts.upload_sources(graph) %}
-        {{ dbt_artifacts.insert_into_metadata_table(
-            database_name=sources.database,
-            schema_name=sources.schema,
-            table_name=sources.identifier,
-            content=content_sources
-            )
-        }}
+        {% set sources_set = [] %}
+        {% for node in graph.nodes.values() | selectattr("resource_type", "equalto", "source") %}
+            {% do sources_set.append(node) %}
+        {% endfor %}
+        {# upload sources in chunks of 500 sources (300 for BigQuery), or less #}
+        {% set upload_limit = 300 if target.type == 'bigquery' else 500 %}
```

Comment:
Is there a reason behind choosing 500 instead of 5000 for the other data warehouses?

Reply:
I'll bump it up.  I couldn't get 500 to work running on my local against BQ.  So I'll leave the BQ limit at 300